web/src/js/components/Header/FlowMenu.tsx

```diff
@@ -24,6 +26,8 @@ export default function FlowMenu(): JSX.Element {
         (state) => state.flows.byId[state.flows.selected[0]],
     );
 
+    const selectedFlows = useAppSelector((state) => state.flows.selected);
```

Comment:
`useAppSelector` is instrumental for caching, see https://redux.js.org/tutorials/fundamentals/part-5-ui-react#reading-state-from-the-store-with-useselector. If we only use `selectedFlows` for `selectedFlows.length > 1` checks, we should only return the boolean from the selector.

Considering this is used repeatedly, I wouldn't mind a `selectHasSingleFlowSelected` helper, but both with and without helper is fine. :)